### PR TITLE
Aggregation filters

### PR DESCRIPTION
**Pull request by angrykoala**
# Description

Add generic filters for aggregations:

```graphql
{
    posts(where: { likesAggregate: { node: { rating: { average: { eq: 3.2 } } } } }) {
        title
    }
}
```

### CODE REVIEW

File: packages/graphql/src/translate/queryAST/factory/FilterFactory.ts [hunk 1]
```diff
@@ -608,6 +613,29 @@ export class FilterFactory {
                 throw new Error(`Invalid operator ${key}`);
         }
     }
+    private parseGenericOperatorForAggregation(key: string): AggregationLogicalOperator {
+        // we convert them to the previous format to keep the same translation logic
+        switch (key) {
+            case "equals":
+            case "eq":
+                return "EQUAL";
+            case "lessThan": // TODO: remove long syntax once tests have been updated
```
Comment (MacondoExpress) [inline packages/graphql/src/translate/queryAST/factory/FilterFactory.ts]: This has already been removed!

File: packages/graphql/src/translate/queryAST/factory/FilterFactory.ts [hunk 2]
```diff
@@ -608,6 +613,29 @@ export class FilterFactory {
                 throw new Error(`Invalid operator ${key}`);
         }
     }
+    private parseGenericOperatorForAggregation(key: string): AggregationLogicalOperator {
```
Comment (MacondoExpress) [inline packages/graphql/src/translate/queryAST/factory/FilterFactory.ts:612]: I believe we can reuse `parseGenericOperator`!

File: packages/graphql/src/translate/queryAST/factory/FilterFactory.ts [hunk 3]
```diff
@@ -919,4 +987,27 @@ export class FilterFactory {
         }
         return targetPoint;
     }
+
+    private parsedGenericAggregationOperator(key: string): AggregationOperator {
```
Comment (MacondoExpress) [inline packages/graphql/src/translate/queryAST/factory/FilterFactory.ts]:
```suggestion
    private parseGenericAggregationOperator(key: string): AggregationOperator {
```

**Review by MacondoExpress — APPROVED** [review]
Left some comments but apart of that it looks great!

File: packages/graphql/src/translate/queryAST/factory/FilterFactory.ts (hunk 2 above)
Comment (angrykoala) [reply packages/graphql/src/translate/queryAST/factory/FilterFactory.ts:612]: These are not the same filters, even if they have the same name, the typings are defined to use one or the other, and I rather keep the nice typings

**Pull request merged**